### PR TITLE
Feature: F3 Landing page

### PR DESCRIPTION
**Pull request by maureenlholland**
## One-line summary

Adds landing page for F3 campaign. This needs to be in production for **Monday Sept. 5** to serve as a live page for paid media QA. **Official go-live is still Tuesday Sept. 6**

## Significant changes and points to review

Most sections have already been code reviewed, the following still need code review:

- parental pro-tip
- passwords section

Cross-browser testing the full page is also needed

**Two outstanding issues will be done in separate tasks:**
- download button JS show/hide logic when user is on Fx and has Fx set as default
- analytics

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/12004

## Testing

http://localhost:8000/en-US/firefox/family

### CODE REVIEW

**Comment by craigcook** [comment]
54 commits... might be a good idea to squash this manually to tidy it up before the merge.

**Review by nathan-barrett — APPROVED** [review]
R+ 🔥 🦊 👪 Really excellent job - I didn't have many comments since I have looked through most of these pages already when we were going through the small PRs - could be useful to have another FE do a few sanity checks.

Did some cross browser testing on browserstack and everything looks great 👌🏻

**Review by craigcook — APPROVED** [review]
This is all amazing work. The page looks great and there's some really clever stuff happening under the hood. It even degrades nicely in old browsers (content stacks without grid support but it's all readable, accessible, and still looks good). Nicely done @maureenlholland and @nathan-barrett!

Just a few comments/questions but no show-stoppers. Happy to merge as-is but I wanted to give you a chance to address them if you want to. A hearty r+wc from me.

**Comment by craigcook** [comment]
Also just making a note that this definitely needs to be squashed and the commit message amended to something short and descriptive, either while merging or squashed manually before it merges.

File: bedrock/firefox/templates/firefox/family/includes/macros.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% macro module_tag(
+  title,
+  anchor
+) -%}
+<a class="c-module-tag" href="#{{ anchor }}">
```
[the remark this answers is not in the document]
Comment (maureenlholland) [reply bedrock/firefox/templates/firefox/family/includes/macros.html:11]: good catch, this appears to be grid related, where the grid item is by default taking up all the width in the column, should be a simple fix with `justify-self: start`

File: media/css/firefox/family/components/_news-ticker.scss [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    &-title,
+    &-item {
+        @include f3.text-body-lg;
+        display: inline;
+        margin-bottom: 0; // protocol override
+    }
+}
+
+// Hide the ticker for reduced motion
+@media screen and (prefers-reduced-motion: reduce) {
```
[the remark this answers is not in the document]
Comment (maureenlholland) [reply media/css/firefox/family/components/_news-ticker.scss:82]: It's a good thought. I'm going to leave as-is for now, but will add to the follow-up issue.

**Pull request merged**